src/Recoil.js

```diff
@@ -29,6 +29,10 @@ export type {
   RecoilValueReadOnly,
 } from './core/Recoil_RecoilValue';
 
+const {unstable_batchedUpdates} = require('./util/ReactBatchedUpdates');
```

Comment:
Do I understand correctly that this module will be resolved differently depending on the build, thus giving us a different default and not requiring setBatch to be called on either platform?

Reply:
Correct. You'll either get `unstable_batchedUpdates` from React DOM or from React Native here depending on the platform. Then we pass that platform-specific batching function to `setBatch`.